design/media_types.go

```diff
@@ -58,6 +58,23 @@ var workItem = a.MediaType("application/vnd.workitem+json", func() {
 	})
 })
 
+// workItem2 is the media type for work items
+var workItem2 = a.MediaType("application/vnd.workitem2+json", func() {
+	a.TypeName("WorkItem2")
+	a.Description("A work item hold field values according to a given field type in JSONAPI form")
+	a.Attribute("links", WorkItemLinks)
+	a.Attribute("data", workItemDataForUpdate)
+	a.Required("links")
+	a.Required("data")
+	a.View("default", func() {
+		a.Attribute("links", func() {
+			a.Attribute("self", d.String)
```

Comment:
thanks, done in https://github.com/almighty/almighty-core/pull/482/commits/d8c08aea9e5f6cc075bed42bc55fb0972ec34018